modules/installation-extend-edge-nodes-aws-local-zones.adoc

```diff
@@ -0,0 +1,93 @@
+
+The `openshift-installer` creates the Machine Set manifest and attaches `node-role.kubernetes.io/edge` and `node-role.kubernetes.io/worker` labels to each edge worker node that is located in a Local Zone subnet.
+
+
+.Prerequisites
+
+* You have an access to the OpenShift CLI (`oc`).
+* You configured an AWS Local Zones environment. 
+* You deployed your cluster om a compatible cloud platform. 
+
+.Procedure
+
+. Create a `deployment` resource YAML file for an edge worker node that operates in a Local Zone subnet. Ensure that you specify the correct tolerations for your edge worker node, so that the scheduler can schedule any pods with matching taints to the cluster.
++
+.Example of a configured portion of the worker manifest file that the VPC uses to run workloads
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment <1>
+metadata:
+  name: <local_zone_name> <2>
+  namespace: <local_zone_namespace> <3>
+spec:
+  selector: 
+    matchLabels:
+      app: <local_zone_application>
+  replicas: 1
+  template:
+    metadata:
+      labels: 
+        app: <local_zone_application>
+        machine.openshift.io/zone-group: ${ZONE_GROUP_NAME} <4>
+    spec:
+      nodeSelector:
+        machine.openshift.io/zone-group: ${ZONE_GROUP_NAME}
+      tolerations: <5>
+      - key: "node-role.kubernetes.io/edge"
+        operator: "Equal"
+        value: ""
+        effect: "NoSchedule"
+      containers:
+        - image: openshift/origin-node
+          command:
+           - "/bin/socat"
+          args:
+            - TCP4-LISTEN:8080,reuseaddr,fork
+            - EXEC:'/bin/bash -c \"printf \\\"HTTP/1.0 200 OK\r\n\r\n\\\"; sed -e \\\"/^\r/q\\\"\"'
+          imagePullPolicy: Always
+          name: echoserver
+          ports:
+            - containerPort: 8080
+----
+<1> `kind`: Defines the `deployment` resource.
+<2> `name`: Specifies the name of the service. 
+<3> `namespace:` Defines the namespace that the `service` resource targets.
+<4> `machine.openshift.io/zone-group`: Defines the group to where a zone belongs. For example, `us-east-1-nyc-1` indicates the region code, `nyc`, and identifiers for a Local Zone.
+<5> `tolerations`: Sets the tag where the scheduler can apply a taint to affected nodes.
+
+
+. Create a `service` resource YAML file for the node.
+.Example of a configured `service` resource for an edge worker node that operates in a Local Zone subnet
+[source,yaml]
+----
+apiVersion: v1
+kind: Service <1>
+metadata:
+  name:  <local_zone_service_name>
+  namespace: <local_zone_name>
+spec:
+  ports:
+    - port: 80
+      targetPort: 8080
+      protocol: TCP
+  type: NodePort
+  selector: <2>
+    app: local-zone-demo-app-nyc-1
+----
+<1> `kind`: Defines the `service` resource.
+<2> `selector:` Specifies the label type applied to managed pods.
```

Comment:
@mtulio .

After a user creates this resource, what happens next? What mechanism does the rest o the tasks? Must a user do more, or does something automatically complete the next steps?

Reply:
As commented on Slack, we can stop here and mention, or put a short paragraph of possibilities (but not required) that users could **look** next, to avoid dup content, of ALB Operator.

Reply:
Thanks for pointing this out. I added the following sentence to callout <5>:

> The `key:` value must match the taint value that you specified in the worker manifest file.

Reply:
Updated.